test/parallel/test-sync-fileread.js

```diff
@@ -1,9 +1,9 @@
 'use strict';
-var common = require('../common');
-var assert = require('assert');
-var path = require('path');
-var fs = require('fs');
+const common = require('../common');
+const assert = require('assert');
+const path = require('path');
+const fs = require('fs');
 
-var fixture = path.join(common.fixturesDir, 'x.txt');
+const fixture = path.join(common.fixturesDir, 'x.txt');
 
-assert.equal('xyz\n', fs.readFileSync(fixture));
+assert.strictEqual('xyz\n', fs.readFileSync(fixture, 'utf8'));
```

Comment:
According to the [documentation](https://nodejs.org/api/fs.html#fs_fs_readfilesync_file_options):

> If the encoding option is specified then this function returns a string. Otherwise it returns a buffer.

So, it seemed to me that the old test with just `assert.equal` was not actually doing much...although I could be wrong?  In any case, I did not see much value in adding another test beyond what was added.

Perhaps I could add an assertion that compares Buffer content of a `readFileSync` call with no encoding?  I am still new to the style and preferences of contributing to the Node.js project, but personally if I was to do such a thing, I would do so in a different PR in an effort to separate *refactoring* tests and *adding* tests.